src/Catalyst.Core.Modules.KeySigner.Tests/UnitTests/KeySignerTests.cs

```diff
@@ -128,5 +128,26 @@ public void KeySigner_Can_Sign_If_Key_Doesnt_Exists_In_Registry_But_There_Is_A_K
 
             Assert.Equal(_signature, actualSignature);
         }
+
+        class CryptoContext : ICryptoContext
+        {
+            readonly ISignature _signature;
+            public CryptoContext(ISignature signature) { _signature = signature; }
+
+            public int PrivateKeyLength { get; }
+            public int PublicKeyLength { get; }
+            public int SignatureLength { get; }
+            public int SignatureContextMaxLength { get; }
+            public IPrivateKey GeneratePrivateKey() { throw new NotImplementedException(); }
+            public IPublicKey GetPublicKeyFromPrivateKey(IPrivateKey privateKey) { throw new NotImplementedException(); }
+            public IPublicKey GetPublicKeyFromBytes(byte[] publicKeyBytes) { throw new NotImplementedException(); }
+            public IPrivateKey GetPrivateKeyFromBytes(byte[] privateKeyBytes) { throw new NotImplementedException(); }
+            public ISignature GetSignatureFromBytes(byte[] signatureBytes, byte[] publicKeyBytes) { throw new NotImplementedException(); }
+            public byte[] ExportPrivateKey(IPrivateKey privateKey) { throw new NotImplementedException(); }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'ExportPrivateKey'.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4569935)